qemu/tests/ovs_qos.py

```diff
@@ -37,7 +37,7 @@ def set_ovs_port_attr(iface, attribute, value):
         """
         cmd = "ovs-vsctl set interface %s %s=%s" % (iface, attribute, value)
         logging.info("execute host command: %s" % cmd)
-        status = process.system(cmd, ignore_status=True)
+        status = process.system(cmd, ignore_status=True).decode()
```

Comment:
@sitoliu process.system() returns an "int" value. Hit below error:

23:22:39 ERROR|   File "/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/qemu/tests/ovs_qos.py", line 40, in set_ovs_port_attr
    status = process.system(cmd, ignore_status=True).decode()
23:22:39 ERROR| AttributeError: 'int' object has no attribute 'decode'